README.md

```diff
@@ -496,12 +496,6 @@ Overrides the default PostgreSQL log directory.
 
 Default value: initdb's default path.
 
-##### `log_line_prefix`
```

Comment:
there's still a `log_line_prefix` param in globals, did you  mean to remove this here?